apluslms_roman/configuration.py

```diff
@@ -20,29 +22,31 @@
     ),),)),
 ))
 
-
 class CourseConfigError(Exception):
     pass
 
-
 def is_legacy_configuration(path):
-    """Checks if path is legacy a-plus-rst-tools course"""
+    # Checks if path is legacy a-plus-rst-tools course
     return isfile(join(path, 'conf.py')) and isfile(join(path, 'Makefile'))
 
+class CourseConfig(Document):
+    name = 'course_config'
+    schema = name
+    version = Version(2, 0)
 
-class CourseConfig:
     @classmethod
     def find_from(cls, path, name=None, prefixes=None):
-        if not name:
-            name = DEFAULT_NAME
+        name = DEFAULT_NAME
```

Comment:
Not sure this changes is supposed to be here. Name argument doesn't do anything now?

Reply:
hm probably messed something up during rebasing